helm-chart/templates/deployment.yaml

```diff
@@ -57,3 +57,65 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app:  {{ template "json-rpc-relay.name" . }}-websocket
```

Comment:
define a `json-rpc-relay.wsname` in the `_helpers.tpl` so you don't have to keep doing this.